### PR TITLE
Added option to export style guide colors using Swift color literals.

### PR DESCRIPTION
**Pull request by AdamSwinden**

### CODE REVIEW

**Comment by artemnovichkov** [comment]
Hi, @AdamSwinden! Thank you for the PR! I'll check it. Could you please attach an example of generated files?

File: package.json [hunk 1]
```diff
@@ -38,11 +38,17 @@
...
-  }],
+  "contributors": [
+    {
+      "name": "Pavel Baybara",
+      "email": "baybara.pavel@gmail.com",
+      "url": "https://github.com/baybara-pavel"
+    },
+    {
+      "name": "Adam Swinden",
+	  "email": "me@swinden.com",
```
Comment (artemnovichkov) [inline package.json]:
```suggestion
	    "email": "me@swinden.com",
```

File: package.json [hunk 2]
```diff
@@ -38,11 +38,17 @@
...
+	  "url": "https://github.com/AdamSwinden"
```
Comment (artemnovichkov) [inline package.json]:
```suggestion
	    "url": "https://github.com/AdamSwinden"
```

**Comment by AdamSwinden** [comment]
I've updated the README to include an example and a description of the new option. Here's an example of the generated files:

```
//
//  UIColor+AppColors.swift
//  Oval - iOS
//
//  Generated by Zepcode on 09/02/2019.
//  Copyright © 2019 __MyCompanyName__. All rights reserved.
//

import UIKit

extension UIColor {

    static let electricBlue = #colorLiteral(red: 0, green: 0.337254902, blue: 1, alpha: 1)
}
```

File: src/templates/color-extension.js [hunk 3]
```diff
@@ -16,7 +17,9 @@ ${
     ${colors.map(color => `static let ${color.name} = ${
       extensionOptions.useCustomColorInitializer
         ? customColorTemplate(color)
-        : colorTemplate(color)
+        : extensionOptions.useColorLiterals
+          ? literalColorTemplate(color)
+          : colorTemplate(color)
```
Comment (artemnovichkov) [inline src/templates/color-extension.js]: Honestly, I don't like this combination of two ternary operators. I guess we can replace it with `Picker` option with three states. What do you think?

Comment (AdamSwinden) [reply src/templates/color-extension.js]: I think you're right — it's not the most readable. `Picker` is probably a better option as the three options we've got at the moment can't all be used together. If we swap how would you expect to see [color-extension.js](https://github.com/artemnovichkov/zepcode/blob/master/src/templates/color-extension.jsrl) formatted?

Comment (artemnovichkov) [reply src/templates/color-extension.js]: I don't know how the picker option works in Zeplin. Anyway, if-else of switch constructions will be a better choice.

Comment (AdamSwinden) [reply src/templates/color-extension.js]: Cool. I'll have a go at it and add it to this pull request.

**Comment by AdamSwinden** [comment]
I've combined the two custom initializer and color literals option into a single initializer style option that lets you pick between default, custom, and literal. The Swift generated is the exact same as before. I've also updated the README with the new option description.

**Comment by artemnovichkov** [comment]
Thank you, @AdamSwinden, looks great! I'll make a new release with your changes.

**Pull request merged**